activepieces/activepieces
# Issue 603 styling autocomplete vars

## What does this PR do?

Addresses #603 

## Type of change



## How should this be tested?

Just go to the builder and add a step that has an auto complete input, and add steps, they should appear with their icons and number, in case of a deleted step "unknown step" shows.
